src/engine/server/sv_init.cpp

```diff
@@ -756,7 +756,7 @@ void SV_Shutdown( const char *finalmsg )
 
 	Cvar_Set( "sv_running", "0" );
 #ifndef BUILD_SERVER
-	NET_Config( true );
+	NET_EnableNetworking( false );
```

Comment:
This probably deserves a comment? Why was this changed?

Reply:
It's not really doing something different. The diff may be confusing because the boolean argument on the left side is `networkingEnabled` and on the right side it is `serverMode` (the responsibility of deciding serverMode was put on the caller).

The `ifndef` stuff is indeed quite confusing, so I added a comment. Thanks for the suggestion.